outdated/modules/httpd.reb

```diff
@@ -225,199 +256,221 @@ sys/make-scheme [
         close?: true
         compress?: false
 
-        render: func [response [string! binary!]][
+        render: method [response [text! binary!]] [
             status: 200
             content: response
         ]
 
-        print: func [response [string!]][
+        print: method [response [text!]] [
             status: 200
             content: response
             type: "text/plain"
         ]
 
-        redirect: [target [url!] /as status [integer!]][
-            status: any [:status 301]
+        redirect: method [target [url!] /as status [integer!]] [
+            status: default [301] ; !!! initialized to 404, so never defaults?
             location: target
         ]
     ]
 
-    transcribe: use [
-        space request-action request-path request-query
-        header-prototype header-feed header-name header-part
-    ][
```

Comment:
Usually-Important-Change-that-doesn't-matter-here: This is trying to make a member function called TRANSCRIBE.  If that member function was of a base object from which derivations were made, merely returning a FUNC/FUNCTION from a USE block would not forward derived references.  You'd have to use METH or METHOD, or BIND it yourself.

In this case, no derived objects are made from the scheme itself, so it doesn't matter yet.  But if it did, this should be a METHOD.  In which case using statics vs. USE would be required.